### DISCUSSION TITLE
Due to GDPR constraints in the EU (Germany)

### DISCUSSION THREAD

**Discussion by gitopandi** [mozilla/hubs#6053] [Hubs Cloud Questions]
**Is your feature request related to a problem? Please describe.**
We (state university) experimenting with Hubs on AWS. For production out of a project context, we need a solution which has GDPR conformity. Mozilla Hubs subscription is only available in Ireland. E-Mail services on AWS only in N. Virginia.

**Describe the solution you'd like**
Mozilla Hubs subscription should be available in the AWS Frankfurt region in Germany. And AWS E-Mail services not in N. Virginal region, also in Frankfurt region.

**Describe alternatives you've considered**
I would be great to have a docker container for using Hubs in our datacenter on premise.

**Additional context**
Thank you for this great tool. We are at the beginning of using 3D environments in educational contexts and Hubs is a great tool for that purpose :-)

**Comment by matthewbcool** [comment]
@gitopandi  Thank you for your request! I'm converting this to a Hubs-Cloud request. 

I also would like to direct you to the #dev-self-hosting channel in our community discord: https://discord.gg/hubs-498741086295031808

See also: https://github.com/mozilla/hubs-compose

Partially related to https://github.com/mozilla/hubs-cloud/issues/222
